app/views/resources/_featured.html.erb

```diff
@@ -1,23 +1,57 @@
-<section>
-  <h2 class="no-margin-top section-title normal"><%= title %></h2>
+<section class="flex flex-col mt-10 px-8">
+  <div class="flex flex-col items-start mb-4">
+    <div class="inline-block ml-24">
+      <h2 class="text-2xl text-gray-700 font-medium m-2"><%= title %></h2>
+      <hr class="h-1 bg-[#063b8d] border-0 rounded-sm my-2 mx-4">
+    </div>
+  </div>
   <% if resources.any? %>
-    <div class="row flex-row" style="padding-top:0px;">
-      <% resources.each do |resource| %>
-        <div class="col-xs-6 col-xs-6-custom">
-          <div class="workshop-box">
-            <%= link_to resource.featured_url, target: :blank do %>
-              <% unless resource.kind == 'Theme' %>
-                <figure class="portrait-img" style="background-image: url(<%= resource.main_image %>)"></figure>
+    <div class="flex flex-wrap justify-center gap-8">
+      <% resources.each do |r| %>
```

Comment:
this is just my opinion and not strongly held, but i find it much more idiomatic (and easier to read) to use the full word for the iterator rather than a single letter. (so, `resources.each do |resource|` rather than `resources.each do |r|`)